open_ce/conda_utils.py

```diff
@@ -102,8 +102,9 @@ def conda_package_info(channels, package):
         _, file_name, rest = entry.partition("file name")
         if file_name:
             entry = yaml.safe_load(file_name + rest)
-            # Convert time string into a timestamp
-            entry["timestamp"] = datetime.timestamp(datetime.strptime(entry["timestamp"], '%Y-%m-%d %H:%M:%S %Z'))
+            # Convert time string into a timestamp (if there is a timestamp)
+            if "timestamp" in entry:
+              entry["timestamp"] = datetime.timestamp(datetime.strptime(entry["timestamp"], '%Y-%m-%d %H:%M:%S %Z'))
```

Comment:
Maybe set `entry["timestamp"] = 0` if there is no timestamp. That way you won't have to handle it anywhere else.

Reply:
Also, please add a test to handle this. You can probably just modify one of the existing samples by removing one of the timestamps.

Reply:
ah, thats a good idea.